### PR TITLE
implemented basic fragment for radius bar

### PR DESCRIPTION
**Pull request by JeremyDezalos**

### CODE REVIEW

**Comment by JeremyDezalos** [comment]
Dear all,

Please ignore code climate duplication problem, those will be resolved when we delete MainActivity.

Thank you.

Have a good day.

Jeremy Dezalos

File: app/src/androidTest/java/com/github/wnder/NavigationActivityTest.java [hunk 1]
```diff
@@ -71,37 +98,19 @@ public void tearDown() {
...
-//        onView(withId(R.id.bottom_navigation)).perform(ViewActions.click(1, 3));
-//        onView(withId(R.id.bottom_navigation)).perform(ViewActions.click(2, 3));
-//        onView(withId(R.id.bottom_navigation)).perform(ViewActions.click(3, 3));
-
-        //All those should do nothing for now
-        assertTrue(true);
+    public void guessButtonShowsSeekbar() {
+        onView(withId(R.id.guess_page)).perform(ViewActions.click());
+        onView(withText("Radius: 5km")).check(matches(isDisplayed()));
+        onView(withId(R.id.navigationToGuessButton)).perform(ViewActions.click());
```
Comment (rkuenzi-epfl) [inline app/src/androidTest/java/com/github/wnder/NavigationActivityTest.java:106]: ViewActions.click() is statically imported yo you can simply use click

File: app/src/main/res/layout/fragment_seekbar.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        app:layout_constraintEnd_toEndOf="parent" />
+
+    <TextView
+        android:id="@+id/radiusTextView"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintBottom_toTopOf="@+id/radiusSeekBar"
+        app:layout_constraintEnd_toEndOf="parent" />
+
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
```
Comment (rkuenzi-epfl) [inline app/src/main/res/layout/fragment_seekbar.xml:26]: Lint warning "Missing `contentDescription` attribute on image"

File: app/src/main/res/layout/fragment_seekbar.xml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        android:id="@+id/navigationToGuessButton"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="16dp"
+        android:layout_marginBottom="16dp"
+        android:clickable="true"
```
Comment (rkuenzi-epfl) [inline app/src/main/res/layout/fragment_seekbar.xml:32]: Lint warning "'clickable' attribute found, please also add 'focusable'"

**Review by rkuenzi-epfl — CHANGES_REQUESTED** [review]
Some lints as always and something you can simplify in test.

**Review by rkuenzi-epfl — APPROVED** [review]

**Review by leonardvaney — CHANGES_REQUESTED** [review]
There is a sick bar, I can move it, click on the button and go to the next activity. The radius in the global user is modified in the listener so it should be working exactly like before.
I just have these two comments + some of the classes have too many unused import and the javadoc is inequal (but don't worry too much about the javadoc).

**Review by leonardvaney — APPROVED** [review]

**Pull request merged**